flocker_bb/builders/maint.py

```diff
@@ -232,7 +232,10 @@ def _destroy_cloud_volumes(self, volumes):
         Unconditionally and irrevocably destroy all of the given cloud volumes.
         """
         for volume in volumes:
-            volume.destroy()
+            try:
+                volume.destroy()
+            except:
+                pass
```

Comment:
Note taken about buildbot not being around forever. Definitely not worth the effort to get them into the step logs.